src/screens/settings/SettingsScreen.tsx

```diff
@@ -49,21 +116,39 @@ const SettingsScreen = ({ navigation }: any) => {
     }
   };
 
-  const getBackHeader = () => (
-    <ButtonHeader onPress={() => navigation.navigate('Home')}>
-      <Appbar.BackAction
-        size={18}
-        style={{ margin: 0 }}
-        onPress={() => navigation.navigate('Home')}
-      />
-      <TextSubtitle>{Text('Go Back')}</TextSubtitle>
-    </ButtonHeader>
-  );
+  const handleRadio = (val: string): void => {
```

Comment:
I moved this code to a helper function called `updateLanguage` in`utils` which handles a string and dictionary update. Read the difference here: https://github.com/calblueprint/siren/pull/79#issue-1242509140

Reply:
```
export const updateLanguage = (
  val: string,
  dictUpdate: any, // JSON DICTIONARY UPDATE FUNCTION  
  stringUpdate: any, // FIREBASE LANGUAGE FIELD UPDATE FUNCTION 
) => {
  if (val === 'Español') {
    dictUpdate(dictionaryList.ES); // DICT TYPE
    stringUpdate('ES'); // STRING TYPE
  }
  if (val === 'Tiếng Việt') {
    dictUpdate(dictionaryList.VIET);
    stringUpdate('VIET');
  }
  if (val === 'English') {
    dictUpdate(dictionaryList.EN);
    stringUpdate('EN');
  }
};```